lib/udap_security_test_kit/registration_success_contents_test.rb

```diff
@@ -48,7 +48,14 @@ class RegistrationSuccessContentsTest < Inferno::Test
 
       original_software_statement = JSON.parse(udap_software_statement_json)
 
-      expected_claims = ['client_name', 'grant_types', 'token_endpoint_auth_method', 'scope']
+      # Scopes received will not exactly match scopes requested if wildcard used
+      # for request (e.g., patient/*.read)
+      # Keep it simple here and just check that scopes are returned, save more
+      # detailed assessment of scope values for SMART-UDAP test kit?
+      assert registration_response.key?('scope'), "Successful registration response must include 'scope' claim"
+      assert registration_response['scope'].present?, 'Scope cannot be blank'
+
+      expected_claims = ['client_name', 'grant_types', 'token_endpoint_auth_method']
```

Comment:
We discussed an exception for `client_name` here too.

Reply:
I updated the code to account for `client_name` and did some refactors to eliminate duplicated code, since the logic/requirements are the same for `scope` and `client_name`.